src/components/Socials/styles.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+
+export const Card = styled.div`
+  display: flex;
+  width: 549px;
+  height: 216px;
+  background-color: #F9F9F9;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 8px;
+`
+
+export const CardText = styled.div`
+  font-family: 'Roboto', sans-serif;
+  text-align: center;
+  font-weight: 700;
+  margin-right: 26px;
```

Comment:
Acho bom trocar o margin-right e o margin-left por:

```suggestion
  padding: 1rem;
```